src/Workleap.OpenApi.MSBuild/LoggerWrapper.cs

```diff
@@ -9,7 +9,7 @@ public LoggerWrapper(TaskLoggingHelper helper)
         this.Helper = helper;
     }
 
-    public TaskLoggingHelper Helper { get; set; }
+    private TaskLoggingHelper Helper { get; }
```

Comment:
Could it be a field instead? I think it was a property because of the first PR, but now I don't see a reason to keep it as a property